applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -121,7 +124,7 @@ def MarkTrailingEdgeElements(self, elem):
                 self.trailing_edge_model_part.Elements.append(elem)
```

Comment:
Why are you avoiding the model part API? I'd suggest to use either the `AddElement` or `AddElements` methods. Note that the second one is more efficient but would require to create an auxiliary array containing the IDs of the elements to be added.

Reply:
Ok will try to use the modelpart API!